Makefile

```diff
@@ -25,34 +28,58 @@ unlink:
 
 .PHONY: lint
 lint:
+	npm --prefix es-bs-base run lint
+# TODO: set up linting for es-ds-components and es-ds-docs
+# 	npm --prefix es-ds-components run lint
+# 	npm --prefix es-ds-docs run lint
+
+.PHONY: legacy-lint
+legacy-lint:
 	npx lerna run lint
 
 .PHONY: test
 test:
+	npm --prefix es-bs-base run test
+# TODO: set up testing for es-ds-components and es-ds-docs
+#	npm --prefix es-ds-components run test
+#	npm --prefix es-ds-docs run test
+
+.PHONY: legacy-test
+legacy-test:
 	npx lerna run test
 
 .PHONY: build
 build:
+	npm --prefix es-bs-base run build
+	npm --prefix es-ds-docs run build
+# es-ds-components does not have a build step
+
+.PHONY: legacy-build
+legacy-build:
 	npx lerna run build
 
 .PHONY: publish
 publish:
+	cd es-bs-base; npm publish --tag alpha
```

Comment:
I see the test 3.0 alpha release in `npm` but not seeing it in the github release [tags](https://github.com/EnergySage/es-ds/tags).  Should it be visible there as well?

Reply:
At the moment, no. I tried to do the absolute minimum necessary to get the NPM publishing working. I'm not entirely clear on whether there's any purpose to the GitHub releases on this repo, but also probably a Tom or Eric question.